types/youtube/index.d.ts

```diff
@@ -518,6 +535,11 @@ declare namespace YT
 		 * Comma separated list of video IDs to play after the URL path's video.
 		 */
 		playlist?: string;
+		
+		/**
+		 * Whether videos play inline or fullscreen in an HTML5 player on iOS. (currently by default, Fullscreen).
+		 */
+		playsinline?: PlaysInline;
```

Comment:
@JoshuaKGoldberg @DanielRosenwasser I'm happy to change it if you guys think the convention is wrong? I've just followed what was there. What I'd put forward originally didn't restrict the value to 1 or a 0.